### DISCUSSION TITLE
Entry Generator Helper's usage

### DISCUSSION THREAD

**Discussion by Harvester62** [camenduru/sd-webui-extension-records#8] [General]
I've noticed that an Entry Generator helper html file has been added in this repository, which I find to be a great idea and a useful tool, but it's unclear to me how to use it, since opening it doesn't show a web page where to fill in the data. 

It might be helpful to further develop it, and provide some instructions on how the developers and users might use it to add and/or update the currently listed Extensions' data.

**Comment by TFWol** [comment]
I actually just threw it together for myself mainly since the repo entries are unwieldy for me to add manually.

You can see it's super barebones. I don't know much html/javascript, but ChatGPT helped 😃 
In order to make it work, the repo would need to have Github Pages activated. I was just saving it on my desktop and opening it in my browser to type things in. 

Preferably, the repo entries themselves should be better, but I can't think of anything simple that works with Github all that well. So I was just going to help add things I personally use to the list to help the repo owner.

**Comment by Harvester62** [comment]
Thank you @TFWol for your quick reply! I do understand, and I appreciate your attempt to simplify the process of adding and updating data here with your "tool", which even if you call it rudimentary is still something, and it is so funny that you used an AI to help with .. AI stuff. :-D 
I don't have either any but basic understanding of Html language, so I can't be very helpful with coding. For sure, having an Html page with that form it would be nice, but it should be also correctly implemented to automatically feed the list, perhaps in the form of a database. I am still thinking loudly.

**Reply by TFWol** [reply]
Yeah, I have in my head what I want, but I'm definitely not the right person to design it. The best method would be a simple table for data storage/entry like you said, then the data could be presented in a sort-able way.

I was hanging out here a bit since @Interpause had mentioned it in his repo and, surprisingly, I didn't see another place that had commit hashes of known working extensions. Info is kind of scattered.

Maybe a list of some kind will be added to Automatic's extension listing eventually.
